Makefile

```diff
@@ -15,7 +15,7 @@ EXTRACTED_TEXT := $(EXTRACT_DIR)/text.po
 JS_TARGET := $(PACKAGE_NAME)/public/js/translations
 TRANSLATIONS_DIR := $(PACKAGE_NAME)/translations
 SOURCES=./setup.py ./$(PACKAGE_NAME)
-BLACK_OPTS = --exclude ${SOURCES}
+BLACK_OPTS = --exclude ${SOURCES} --line-length 100
```

Comment:
Same comment here, we usually use 120.

Reply:
Done: https://github.com/eduNEXT/xblock-discussion-grading/commit/b7fbbc27a38d4555abbdab7817a87c7081ce496c